src/pages/api-release-notes.tsx

```diff
@@ -98,7 +113,7 @@ export default function ReleaseNotesListComponent(
   `)
   return (
     <Layout>
-      <Container maxWidth="md">
+      <Container maxWidth="lg">
```

Comment:
Should we try and keep all the element wrappers consistent with the main 'DocBody' component, utilizing the `marginLeft: theme.spacing(9)`, and `marginRight: theme.spacing(DrawerWidthSpacingLg)`.  That way, the content is contained exactly, and it allows us to reduce the amount of nested elements.

Reply:
would that be instead of using this maxWidth option? Sorry im a newbbbb

Reply:
Well, it would just involve removing the `Container` component, as well as  both `Grid` components, and using a `Box` component to give the margin~s~ (we only need `marginLeft` here, since the `RightMenu` doesn't show up on this page).  Then you would give it a `maxWidth` on that Box if you didn't want it spanning the whole page.  But that way it's 1 wrapper, not 3.

Reply:
got it! i'll take a crack at that